ldes-server-domain/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/entities/LdesMemberTest.java

```diff
@@ -41,11 +40,11 @@ void when_TreeMemberStatementIsReplaced_TreeMemberStatementHasADifferentSubject(
     @DisplayName("Verify retrieving of member id from LdesMember")
     void when_TreeMemberStatementIsAvailableInModel_LdesMemberId() throws IOException {
         String ldesMemberString = FileUtils.readFileToString(ResourceUtils.getFile("classpath:example-ldes-member.nq"), StandardCharsets.UTF_8);
-        LdesMember ldesMember = new LdesMember(createModel(ldesMemberString, Lang.NQUADS));
-        assertEquals("https://private-api.gipod.beta-vlaanderen.be/api/v1/mobility-hindrances/10228622/483",ldesMember.getLdesMemberId(MEMBER_TYPE));
+        LdesMember ldesMember = new LdesMember("some_id", createModel(ldesMemberString, Lang.NQUADS));
+        assertEquals("some_id", ldesMember.getLdesMemberId());
```

Comment:
This Id was aligned with the member in the quad file.. The fact that you now pass the Id directly in the line above makes this test rather "dumb" in my opinion since this used to validate the Id extraction.